lib/Ast.ml

```diff
@@ -2269,6 +2269,13 @@ end = struct
     ||
     match (ctx, exp) with
     | Str {pstr_desc= Pstr_eval _; _}, _ -> false
+    | ( Str
+          { pstr_desc=
+              Pstr_value
+                (Nonrecursive, [{pvb_pat= {ppat_desc= Ppat_any; _}; _}])
+          ; _ }
+      , _ ) ->
+        false
```

Comment:
I wonder if we can remove parentheses in all `Str _, _` cases ? `test_branch` does not catch any bug.

Reply:
Shouldn't it find cases where there were extra parens?

Reply:
In any case, I don't have enough experience to determine if it's risky or not, but if it is we can first extend to let _, and all lets.

Reply:
I don't have any risky case in mind either, but I agree we can start with the current state of the PR and then extend later.

Reply:
> Shouldn't it find cases where there were extra parens?

This is to work around the next pattern (`... when has_trailing_attributes_exp ... -> true`). So it only removes parentheses in cases like `let a = (some expr) [@a]`, which are rare.